.circleci/config.yml

```diff
@@ -17,6 +17,10 @@ executors:
     docker:
       - image: circleci/node:lts
 
+  native:
+    docker:
+      - image: circleci/openjdk:8-jdk
```

Comment:
Running native code shouldn't have any dependency on the JVM. If it does, I suspect we're doing something wrong.

Reply:
Indeed, I thought it would require some call to gradle, but no 😄